### ISSUE TITLE
`cargo dybuk`?

### ISSUE THREAD

**Issue by ticki** [kbknapp/cargo-extras#11]
I wrote https://github.com/ticki/dybuk a while ago, which essentially clean up errors. And I thought this project was a nice collection of cargo tools.

Is there any possibility of adding `cargo dybuk` for directly piping the output through dybuk?

**Comment by kbknapp** [comment]
Absolutely! Would you like to create `cargo-dybuk` project, or just boulder dybuk as is and rename the binary to `cargo-dybuk` from this repo? (Haven't tested the latter, but I'm assuming it'd work just fine).

**Comment by ticki** [comment]
The second would unfortunately not work, since dybuk is mainly targeting rust but also supports other languages, as such you have to pipe the output of rustc (cargo) through dybuk... Will write a wrapper doing this (cargo dybuk).

**Comment by kbknapp** [comment]
Sounds great, once you've got it read let me know and I'll include and put out a new release! :+1:
